packages/react-native/ReactCommon/react/renderer/graphics/Transform.h

```diff
@@ -47,9 +48,15 @@ enum class TransformOperationType {
 
 struct TransformOperation {
   TransformOperationType type;
-  Float x;
-  Float y;
-  Float z;
+  ValueUnit x;
+  ValueUnit y;
+  ValueUnit z;
+  bool operator==(const TransformOperation& other) const {
+    return type == other.type && x == other.x && y == other.y && z == other.z;
+  }
+  bool operator!=(const TransformOperation& other) const {
+    return !(*this == other);
+  }
```

Comment:
nits:
1. C++ 20 lets you default this operator
2. C++ 20 will automatically define `!=` if you define `==`
```suggestion
  bool operator==(const TransformOperation& other) const = default;
```